app/ante/eth.go

```diff
@@ -309,7 +309,10 @@ func (ctd CanTransferDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 		}
 
 		stateDB := statedb.New(ctx, ctd.evmKeeper, statedb.NewEmptyTxConfig(common.BytesToHash(ctx.HeaderHash())))
-		evm := ctd.evmKeeper.NewEVM(ctx, coreMsg, cfg, evmtypes.NewNoOpTracer(), stateDB)
+		evm, err := ctd.evmKeeper.NewEVM(ctx, coreMsg, cfg, evmtypes.NewNoOpTracer(), stateDB)
+		if err != nil {
+			return ctx, err
```

Comment:
Looks like it need a wrap with errorsmod.

Reply:
https://github.com/xpladev/ethermint/pull/14/commits/57497dde9ee668ed5e95bc3bec40fae639c1b1a4